specifications/M1/credential.issuance.protocol.md

```diff
@@ -0,0 +1,313 @@
+
+> Note: Properties mapped to the _Open ID for Verifiable Credential Issuance_ specification are defined in
+>
+the [Credential Issuer Metadata](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#section-10.2.3.1)
+> section.
+
+The following is a non-normative example of a `credentials object`:
+
+```json
+{
+  "format": "ldp_vc",
+  "@context": {
+    "ssi": "https://w3id.org/edc/v0.8/",
+    "odrl": "https://www.w3.org/ns/odrl.jsonld/",
+    "bindingMethods": "ssi:bindingMethods",
+    "cryptographicSuites": "ssi:cryptographicSuites",
+    "issuancePolicy": "ssi:issuancePolicy",
+    "offerReason": "ssi:offerReason",
+    "credentialSubject": "ssi:credentialSubject",
+    "types": "ssi:types"
+  },
+  "types": [
+    "VerifiableCredential",
+    "CompanyCredential"
+  ],
+  "offerReason": "reissue",
+  "bindingMethods": [
+    "did:web"
+  ],
+  "cryptographicSuites": [
+    "JsonWebKey2020"
+  ],
+  "issuancePolicy": {
+    "permission": [
+      {
+        "action": "use",
+        "constraint": {
+          "and": [
+            {
+              "leftOperand": "ssi:CredentialPrereq",
+              "operator": "eq",
+              "rightOperand": "active"
+            }
+          ]
+        }
+      }
+    ]
+  }
+}
+```
+
+> TODO: Define SSI context and include term alias that are shown above. Also, need to define a way to
+> associate `issuancePolicy` to an ODRL policy so that Json-Ld expansion is performed properly on sub-nodes.
+
+# 5. Issuer Metadata endpoint
+
+A credential issuer MUST support the Issuer Metadata endpoint using the HTTPS scheme and the `GET method`. The URL of
+the endpoint is the base issuer url with the appended path `/.well-known/vci`.
+
+The response is a JSON object with the following properties:
+
+- `credentialIssuer`: REQUIRED. A unique identifier of the issuer, for example, a DID.
+- `credentialsSupported`: OPTIONAL. A Json Array containing a list of Json Objects with properties corresponding
+  to [Credential Offer Parameters](#credential-offer-parameters).
+
+```json
+{
+  "credentialIssuer": "did:web:issuer-url",
+  "credentialsSupported": [
+    {
+      "types": [
+        "VerifiableCredential",
+        "CompanyCredential"
+      ],
+      "offerReason": "reissue",
+      "bindingMethods": [
+        "did:web"
+      ],
+      "cryptographicSuites": [
+        "JsonWebKey2020"
+      ],
+      "issuancePolicy": {
+        "permission": [
+          {
+            "action": "use",
+            "constraint": {
+              "and": [
+                {
+                  "leftOperand": "ssi:CredentialPrereq",
+                  "operator": "eq",
+                  "rightOperand": "active"
+                }
+              ]
+            }
+          }
+        ]
+      }
+    }
+  ]
+}
+
+```
+
+# 6. Credential Request Status Endpoint
+
+The issuer MUST provide an `HTTPS GET` endpoint for retrieving the status of a credential at the base issuer url with
```

Comment:
... provide an HTTP `GET` endpoint with the relative path `/requests/<request-id>`. The endpoint must be access controlled.

Reply:
HTTP is explicitly prohibited.

Reply:
> Hypertext Transfer Protocol Secure (HTTPS) is an extension of the [Hypertext Transfer Protocol](https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol) (HTTP)
HTTP only refers to the method definitions and not the plain text protocol usage.

See also: https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods

Reply:
Yes, of course, but it should be explicit HTTPS is used. We can open another issue and update all of the documents in unison.

Reply:
It is explicit, you've stated it at the beginning of the section. Let's open an issue and update all docs.

Reply:
OK but it should not state only `HTTP` as that will be in conflict to the section where it is explicitly stated.